test/cli/autocomplete_test.py

```diff
@@ -8,7 +8,7 @@
     "incomplete,expected",
     [
         ["an", ["ansi"]],
-        ["s", ["snowflake", "soql", "sparksql", "sqlite"]],
+        ["s", ["sparksql", "snowflake", "soql", "sparksql", "sqlite"]],
```

Comment:
Additional instance of sparksql is generated by the "databricks" dialect

Reply:
I think you can fix this by changing this:

https://github.com/sqlfluff/sqlfluff/blob/20f065c846245307bad853b91b557f5619ac492a/src/sqlfluff/cli/autocomplete.py#L21

to this:

```py
    dialect_names = [e.label for e in list_dialects()]
```

Which also allows you to add back your test to `test/cli/autocomplete_test.py`:

```py
        ["d", ["databricks", "db2"]],
```